pandas/tests/test_downstream.py

```diff
@@ -277,3 +286,70 @@ def __radd__(self, other):
 
     assert right.__add__(left) is NotImplemented
     assert right + left is left
+
+
+@pytest.fixture(
+    params=[
+        "memoryview",
+        "array",
+        pytest.param("dask", marks=td.skip_if_no("dask.array")),
+        pytest.param("xarray", marks=td.skip_if_no("xarray")),
```

Comment:
not for this PR, but maybe we should just say these types aren't supported in the relevant constructors?

Reply:
Yeah makes sense. Users should ideally be passing pandas or native python objects